www/templates/layouts/base.html

```diff
@@ -97,6 +97,16 @@
         gtag('config', 'G-1C1KF63NYM');
     </script>
 {% endif %}
+<script
```

Comment:
can we avoid using jquery just to get an element / tottle it?

Reply:
Hi @iseletsk We can not avoid using jquey here because we need to get the click event on the buttons and for that we need to do that in jquery I don't think there is any other option for this.